src/modules/attributes.ts

```diff
@@ -26,28 +26,31 @@ function updateAttrs(oldVnode: VNode, vnode: VNode): void {
   if (oldAttrs === attrs) return;
   oldAttrs = oldAttrs || {};
   attrs = attrs || {};
+  const lowAttrs: any = {};
 
   // update modified attributes, add new attributes
   for (key in attrs) {
     cur = attrs[key];
-    old = oldAttrs[key];
+    const lowkey = key.toLowerCase();
```

Comment:
why lowercasing the keys in `attrs` when attribute keys are suppose to be in lowercase in the first place? snabbdom is a low-level API, and whoever relies on `attrs` must ensure that attribute keys (per spec) are lowercased. Seems to me that we are slowing down the critical path to accomodate a very edge case in which someone uses snabbdom incorrectly.